srcdeps-core/src/main/resources/gradle/settings/srcdeps-model-transformer.gradle

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2015-2017 Maven Source Dependencies
+ * Plugin contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Sets an {@code afterEvaluate} action to each project in this source tree that does the following:
+ *
+ * (1) It sets project versions to the version specified in {@code SrcdepsInner.version}
+ * (2) It it applies the {@code maven} plugin unless the plugin is applied already
+ */
+ gradle.projectsLoaded {
+
+    gradle.rootProject.properties['allprojects'].each {
+        it.afterEvaluate { project ->
+            def groupId = project.group
+            def artifactId = project.name // unfortunately this will not be accurate in all cases; improvements welcome!
+            def version = project.version
+
+            logger.error("srcdeps is processing ${groupId}:${artifactId}:${version}")
+            if (srcdepsInner.gavSet.contains(groupId, artifactId, version)) {
+                def plugins = project.plugins;
+                if (!plugins.hasPlugin('maven')) {
+                    logger.error("srcdeps is adding maven plugin to ${groupId}:${artifactId}")
```

Comment:
I think you answered that before already, but I forgot the answer: will this print anything to the target's console?